pkg/network/tracer/connection/ebpfless_tracer.go

```diff
@@ -285,6 +291,9 @@ func (t *ebpfLessTracer) GetConnections(buffer *network.ConnectionBuffer, filter
 	log.Trace(t.conns)
 	conns := make([]network.ConnectionStats, 0, len(t.conns))
 	for _, c := range t.conns {
+		if c.Type == network.TCP && !t.tcp.HasConnEstablished(c) {
```

Comment:
Could this possibly skip any closed connections?

Reply:
We discussed this in person, the tracer needs to start using the function passed into Start() to properly handle closed connections. I will do this in an upcoming PR

Reply:
I've re-done this in [[ebpfless] Separate pending connection map, handle closed connections](https://github.com/DataDog/datadog-agent/pull/32158) which is stacked on this PR - that PR removes `HasConnEstablished` entirely